test/DurableTask.SqlServer.AzureFunctions.Tests/IntegrationTestBase.cs

```diff
@@ -139,9 +165,22 @@ protected Task CallFunctionAsync(string name, IDictionary<string, object?>? args
             return status;
         }
 
+
+        protected async Task StartOrchestrationWithoutWaitingAsync(
```

Comment:
Why is this new method needed? I see it's used in the new test you added, but I don't understand why it's preferred over the existing `StartOrchestrationAsync`. If you don't strictly need the "WithoutWaiting" behavior, then I'd prefer we use the existing method, which provides better determinism.

Reply:
The client WaitForStartAsync is failing due to the fact is waiting for another status than pending but as it's on another taskhubname the other one is never change from pending I will fix the first case

Reply:
If you prefer another name I can change it

Reply:
Why would it be on another task hub name? The client that starts the new orchestration is the same one that's waiting for it to complete.

Reply:
Yes the client but not the worker

Reply:
oh, it's because they are being started on a task hub that doesn't have a corresponding worker, right? I that case, this makes sense.